utils/contentful.ts

```diff
@@ -46,6 +46,9 @@ export type ContentfulSynth = {
   description: string;
   logo?: Entry<ContentfulLogo>;
   mintmanage: string;
+  externalApi?: string;
+  name?: string;
+  externalUrl?: string;
```

Comment:
These are the new optional Contentful fields

Reply:
i wondering if while you are in there we could add an optional `isExpired` field? the reason i ask is because its the only data on the listing page which is still dependent on the api. we have to fetch all the data about the contracts in listing page to know what expiration date is, or if its expired. If we have this feature, it has no effect on current logic if not set, but if set it can override from contentful if we need to.  wdyt?

Reply:
I'm in favor of doing it.

Reply:
we discussed offline, lets leave it for now